### PR TITLE
create process summary

### PR DESCRIPTION
**Pull request by lindsey-w**

### CODE REVIEW

File: wintappy/etlutils/dbhelpers.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    args = get_configs(parser, argv)
+
+    # Set path and name for helperdb file.
+    dbname = args.NAME if args.NAME else args.AGGLEVEL + ".db"
+    dbpath = f"{args.PATH}" if args.PATH else f"{args.DATASET}{os.sep}dbhelpers"
+    if not os.path.exists(dbpath):
+        os.makedirs(dbpath)
+        logging.debug(f"created folder: {dbpath} ")
+
+    print(f"Writing helperdb to: {dbpath}{os.sep}{dbname}")
```
Comment (lindsey-w) [inline wintappy/etlutils/dbhelpers.py:39]: use the logger? (for all the print statements in here)

File: wintappy/etlutils/download_from_s3.py [hunk 2]
```diff
@@ -281,29 +281,18 @@ def parse_s3_metadata(files, local_path, uploadedDPK, uploadedHPK, event_type):
...
-    args.update({k: v for k, v in vars(options).items() if v is not None})
-
-    try:
-        logging.getLogger().setLevel(args.LOG_LEVEL)
-    except ValueError:
-        logging.error("Invalid log level: {}".format(args.LOG_LEVEL))
-        sys.exit(1)
+    args = get_configs(parser, argv)
 
+    # Move into get_configs?
```
Comment (lindsey-w) [inline wintappy/etlutils/download_from_s3.py:295]: yea, that sounds like a good idea

File: wintappy/etlutils/rawtostdview.py [hunk 3]
```diff
@@ -1,52 +1,39 @@
...
 from importlib.resources import files as resource_files
 
-from wintappy.config import get_config
+from wintappy.config import get_configs, print_config
 from wintappy.datautils import rawutil as ru
 from wintappy.etlutils.utils import configure_basic_logging
 
+# def create_networkx_view(con, dataset):
+#    sql = f"create or replace view labels_networkx as select * from read_json_auto('{dataset}/labels/networkx/*.json', filename=true)"
+#    con.execute(sql)
```
Comment (lindsey-w) [inline wintappy/etlutils/rawtostdview.py:10]: this moved?

File: wintappy/etlutils/dbhelpers.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    parser.add_argument(
+        "-n",
+        "--name",
+        help="Name for the duckdb created. Defaults to [agglevel].duckdb",
+    )
+    parser.add_argument(
+        "-p",
+        "--path",
+        help="Path for the duckdb created. Defaults to [dataset]/[dbhelper]/[agglevel].duckdb",
+    )
```
Comment (lindsey-w) [inline wintappy/etlutils/dbhelpers.py:29]: do we want to add examples in the settings file?

File: wintappy/etlutils/download_from_s3.py [hunk 5]
```diff
@@ -281,29 +281,18 @@ def parse_s3_metadata(files, local_path, uploadedDPK, uploadedHPK, event_type):
 def main(argv=None) -> None:
     configure_basic_logging()
     parser = argparse.ArgumentParser(
-        prog="downloadFromS3.py", description="Download Wintap files from S3"
+        prog="download_from_S3.py", description="Download Wintap files from S3"
```
Comment (lindsey-w) [inline wintappy/etlutils/download_from_s3.py:284]:
```suggestion
        prog="download_from_s3.py", description="Download Wintap files from S3"
```

File: wintappy/etlutils/dbhelpers.py (hunk 1 above)
Comment (grantjllnl) [reply wintappy/etlutils/dbhelpers.py:39]: Good point, changed to logger.info.

File: wintappy/etlutils/download_from_s3.py (hunk 5 above)
Comment (grantjllnl) [reply wintappy/etlutils/download_from_s3.py:284]: I went further and removed underscores to match the other programs.

File: wintappy/etlutils/rawtorolling.py [hunk 6]
```diff
@@ -30,32 +31,16 @@ def main(argv=None) -> None:
...
+    args = get_configs(parser, argv)
 
-    print_config(args)
-
-    start_date, end_date = get_date_range(args.START, args.END)
+    start_date, end_date = get_date_range(
+        args.START, args.END, data_set_path=args.DATASET
+    )
 
+    print(f"Processing {start_date} to {end_date}")
```
Comment (lindsey-w) [inline wintappy/etlutils/rawtorolling.py:43]:
```suggestion
    logging.info(f"Processing {start_date} to {end_date}")
```

**Pull request merged**